### DISCUSSION TITLE
论emoji合成

### DISCUSSION THREAD

**Discussion by WangZhaoming114514** [MelodyYuuka/YuukaBot-docs#14] [General]
你uuz bot在某些群聊会被别有用心之人挂有🐻与🤡😈😎等emoji合成敏感表情，建议不支持融合此类表情

**Comment by MelodyYuuka** [comment]
不是，这只是个熊而已啊，不要太联想

**Reply by WangZhaoming114514** [reply]
问题是我看到真有人做，不然我也不会特意上GitHub说![Image_1721815053484.png](https://github.com/user-attachments/assets/7ab337d2-d381-4085-8428-f8b22d84a859)

**Reply by MelodyYuuka** [reply]
其实不是很像，太能联想了（
